addons/sale_stock/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,21 @@
+---Fields in module 'sale_stock'---
+sale_stock   / stock.move               / to_refund_so (boolean)        : NEW
```

Comment:
Please marked as default this field, as if you come from v9, you expect that all the already made moves are refundable.